src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -80,6 +83,9 @@ def __init__(
         :param interface: python-can interface modules used
         :param channel: the can interface name
         :param state: BusState of the channel
+        :param trace_path: path to write the trace
+        :param trace_size: maximum size of the trace in (Mb)
```

Comment:
```suggestion
        :param trace_size: maximum size of the trace (in MB)
```
Maybe also specify the possible values (1 to 100)